neptune/alpha/internal/threading/daemon.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, sleep_time: float):
         self._sleep_time = sleep_time
         self._interrupted = False
         self._event = threading.Event()
+        self._is_running = False
```

Comment:
I would created 3-state enum instead 2 booleans interrupted and is_running but won't force it if you don't have time for it